lib/puppet_x/chocolatey/chocolatey_common.rb

```diff
@@ -30,12 +30,12 @@ def chocolatey_command
           #   it installs
           chocoInstallPath = PuppetX::Chocolatey::ChocolateyInstall.install_path
 
-          chocopath =  (chocoInstallPath if (chocoInstallPath && file_exists?("#{chocoInstallPath}\\bin\\choco.exe"))) ||
-              ('C:\ProgramData\chocolatey' if file_exists?('C:\ProgramData\chocolatey\bin\choco.exe')) ||
-              ('C:\Chocolatey' if file_exists?('C:\Chocolatey\bin\choco.exe')) ||
+          chocopath =  (chocoInstallPath if (chocoInstallPath && file_exists?("#{chocoInstallPath}\\choco.exe"))) ||
```

Comment:
Not if you support only 0.9.9+ (which this module only supports that). If by chance there is the scenario you describe, that shim won't work because `bin\choco.exe` (the shim) is looking for `choco.exe`. HTH